`tests/rustdoc-gui/notable-trait.goml` is sometimes flaky

On `x86_64-gnu-tools`:

```
======== tests/rustdoc-gui/notable-trait.goml ========
[ERROR] line 255: expected 0 elements, found 1: for command `assert-count: ("//*[@class='tooltip popover']", 0)`
    at <file:///checkout/obj/build/x86_64-unknown-linux-gnu/test/rustdoc-gui/doc/test_docs/struct.NotableStructWithLongName.html#method.create_an_iterator_from_read>
======== tests/rustdoc-gui/search-result-display.goml ========
[WARNING] line 39: Delta is 0 for "x", maybe try to use `compare-elements-position` instead?
<= doc-ui tests done: 143 succeeded, 1 failed, 0 filtered out
Error: ()
Bootstrap failed while executing `test tests/rustdoc-gui --stage 2 --test-args --jobs 1`
```

- PR: https://github.com/rust-lang/rust/pull/150988#issuecomment-3737979419
- Log: https://triage.rust-lang.org/gha-logs/rust-lang/rust/60067226053

Unfortunately I can't reproduce this at all locally :(

Happened again in

```
======== tests/rustdoc-gui/notable-trait.goml ========

[ERROR] line 254
    at `tests/rustdoc-gui/utils.goml` line 34: Error: The following CSS properties still don't match: [expected `block` for key `display`, found `none`]: for command `wait-for-css: ("#settings", {"display": "block"})`
    at <file:///checkout/obj/build/x86_64-unknown-linux-gnu/test/rustdoc-gui/doc/test_docs/struct.NotableStructWithLongName.html#method.create_an_iterator_from_read>

======== tests/rustdoc-gui/search-result-display.goml ========

[WARNING] line 39: Delta is 0 for "x", maybe try to use `compare-elements-position` instead?
```

https://github.com/rust-lang/rust/pull/150541#issuecomment-3742289591

--- a/tests/rustdoc-gui/notable-trait.goml
+++ b/tests/rustdoc-gui/notable-trait.goml
@@ -250,7 +250,7 @@ set-window-size: (1100, 600)
 reload:
 assert-count: ("//*[@class='tooltip popover']", 0)
 click: "//*[@id='method.create_an_iterator_from_read']//*[@class='tooltip']"
-assert-count: ("//*[@class='tooltip popover']", 1)
+wait-for-count: ("//*[@class='tooltip popover']", 1)
 call-function: ("open-settings-menu", {})
-assert-count: ("//*[@class='tooltip popover']", 0)
+wait-for-count: ("//*[@class='tooltip popover']", 0)
 assert-false: "#method\.create_an_iterator_from_read .tooltip:focus"

